recipes/date/all/conanfile.py

```diff
@@ -1,29 +1,91 @@
 import os
-from conans import ConanFile, tools
+from conans import ConanFile, CMake, tools
 
 
 class DateConan(ConanFile):
     name = "date"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/HowardHinnant/date"
     description = "A date and time library based on the C++11/14/17 <chrono> header"
-    topics = ("date", "datetime", "timezone", "calendar", "time", "iana-database")
+    topics = ("date", "datetime", "timezone",
+              "calendar", "time", "iana-database")
     license = "MIT"
-    exports_sources = ["patches/*"]
-    no_copy_source = True
+    exports_sources = ["patches/*", "CMakeLists.txt"]
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "use_system_tz_db": [True, False],
+               "use_tz_db_in_dot": [True, False]}
+    default_options = {"shared": False,
+                       "fPIC": True,
+                       "use_system_tz_db": True,
```

Comment:
By default, this option is false. Follow the same behavior from the upstream.

Reply:
Changed, nice spot.
however it will then by default add curl dependency
in bincrafters it was true by default:
https://github.com/bincrafters/conan-date/blob/testing/2.4.1/conanfile.py#L18

Reply:
probably because it helps more all users.

Reply:
Applied. resolving.